app/src/main/java/lt/vilnius/tvarkau/activity/LoginActivity.kt

```diff
@@ -84,6 +89,7 @@ class LoginActivity : BaseActivity() {
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
         when (requestCode) {
             RC_SIGN_IN -> viewModel.signInWithGoogle(data)
+            RC_VIISP_SIGN_IN -> viewModel.signInWithViisp(data)
```

Comment:
Don't we care about `resultCode` ? I think we should call this only on `RESULT_OK`.

Reply:
Well now we don't set any other results. Have any ideas when we should set other result codes?

Reply:
Doesn't matter. It should be ok as we don't have any other `setResult` inside activity.